conf/layer.conf

```diff
@@ -0,0 +1,12 @@
+# We have a conf and classes directory, append to BBPATH
```

Comment:
A short README would be great. Explaining the emta purpose and linking to the safeboot repo or documentation.

Reply:
Added